src/CspNijmegenStage.ts

```diff
@@ -13,10 +15,18 @@ export class CspNijmegenStage extends Stage {
 
     // Csp zone related stuff
     // Iam delegation policies
-    new CspNijmegenStack(this, 'stack', {
+    const csp = new CspNijmegenStack(this, 'stack', {
       env: props.cspRootEnvironment,
       sandbox: props.sandbox,
     });
 
+    // KMS key for dnssec (must be in us-east-1)
+    const dnssec = new DnsSecStack(this, 'dnssec-stack', {
+      alias: 'gemeente-nijmegen/dnssec',
```

Comment:
moet de alias niet specifieker? Als we toch nog een dnssec kms key voor iets anders nodig hebben kom je denk ik in de knoei met naamgeving.

Reply:
misschien een niveau 'defaults' toevoegen voor standaard gedeelde resources? `gemeente-nijmegen/defaults/dnssec' of eventueel andersom? `gemeente-nijmegen/dnssec/default`

Reply:
dat is denk ik wel een goede oplossing

Reply:
Voor de account subdomeinen (accp.csp-nijmegen.nl, sandbox.csp-nijmegen.nl) gebruikt ik `/gemeente-nijmegen/account/hostedzone/name` dus misschien `/gemeente-nijmegen/account/dnssec/kmskey`?

Reply:
naamgeving.... altijd fijn.... het is hier de alias van de kmskey toch? `/gemeente-nijmegen/account/dnssec/kmskey` klinkt meer als een parameter naam. Iets als `gemeente-nijmegen/dnssec/default` (suggestie van Joost) lijkt me dan logischer